ConditionalOpetators/Stock/Stock.java

```diff
@@ -0,0 +1,44 @@
+package tasks.conditionaloperator;
+
+    /**
+    https://moodle.jrr.by/practice/141
+    */
+
+public class Stock {
+
+    String companyName;
+    int actualPrice;
+    int minPrice;
+    int maxPrice;

```

Comment:
are you sure that you need default access modificator?
How about `private` ? Read some infromation about `incapsulation` one of the OOP principle